Hunted Mobile/Hunted Mobile/Hunted Mobile/Model/Location.cs

```diff
@@ -23,6 +23,17 @@ public class Location {
 
         }
 
+        public Location(string commaSeparatedLatitudeLongitude) {
+            string[] split = commaSeparatedLatitudeLongitude.Split(',');
+            try {
+                Latitude = double.Parse(split[0]);
+                Longitude = double.Parse(split[1]);
+            }
+            catch(Exception) {
+
```

Comment:
empty catch blok is wel bad practice. Zou fijn zijn als je dit nog even gauw kan fixen